fresnel/geometry.py

```diff
@@ -183,6 +183,157 @@ def radius(self):
     def color(self):
         return util.array(self._geometry.getColorBuffer(), geom=self)
 
+
+class Box(Cylinder):
+    R""" Box geometry.
+
+    Generate a "box" shape.
+
+    Args:
+        scene (:py:class:`fresnel.Scene`): Add the geometry to this scene
+        box (`numpy.ndarray` or `array_like`): (``1x1, 1x3, or 1x6`` : ``float32``) Assumes 1x1 is cubic, 1x3 is orthorhombic, and 1x6 is triclinic.
+        radius (`numpy.ndarray` or `array_like`): (``N`` : ``float32``) - Radius of box edges.
+        color (`numpy.ndarray` or `array_like`): (``Nx2x3`` : ``float32``) - Color of box edges.
+
+    .. seealso::
+
+        :doc:`examples/01-Primitives/05-Box-geometry`
+            Tutorial: defining and setting box geometry properties
+        :doc:`examples/02-Advanced-topics/05-GSD-visualization`
+            Tutorial: Visualizing GSD files
+
+
+    Note:
+        The constructor arguments ``radius``, and ``color`` are optional. If you do not provide them,
+        they are initialized to 0.5 and black, respectively.
+
+    .. hint::
+        Avoid costly memory allocations and type conversions by specifying primitive properties in the appropriate
+        numpy array type.
+
+    Attributes:
+        points (:py:class:`fresnel.util.array`): Read or modify the box.
```

Comment:
I wouldn't expose the points attribute (at least don't make it settable). I would make a custom getter/setter for `self.box` as a `@property` that calls `_generate_points()`.

Reply:
Thanks for the suggestion! We've made a getter and setter for the box attribute. Modifying the points attribute is already disabled, but the user can still manually change them using `box.points[:] = 0` is there a way to override the freud.util.array.__setitem__ for this or should we just leave it be?

Reply:
`Box` is-a `Cylinder`, so it should inherit these properties and users should be able to modify them if they wish to. Maybe a user really wants the bottom of the box to be red and the top blue :) The `box` property should document that it sets `points`.